### ISSUE TITLE
provide more details about the position at which the difference happens

### ISSUE THREAD

**Issue by tyrcho** [WriteThemFirst/approvals-java#81]
For instance, this error is difficult to fix:

```
[info]   java.lang.AssertionError: expected: <Success(
[info]   Event(
[info]     "T\ufffdrmilihus Winterthur",
[info]     "https://www.mtgtop8.com/event?e=18427",
[info]     8,
[info]     Sun Jan 28 00:00:00 CET 2018,
[info]     Highlander,
[info]     None
[info]   )
[info] )
[info] > but was: <Success(
[info]   Event(
[info]     "T\ufffdrmilihus Winterthur",
[info]     "https://www.mtgtop8.com/event?e=18427",
[info]     8,
[info]     Sun Jan 28 00:00:00 UTC 2018,
[info]     Highlander,
[info]     None
[info]   )
[info] )>
```

**Comment by tyrcho** [comment]
the position of the error would help + some context (a few characters before / after)

**Comment by tyrcho** [comment]
fixed in 1ccbe00b27df502f590fc670b83798f9b1b1fbf9
